packages/jellyfin-server/build.sh

```diff
@@ -0,0 +1,267 @@
+TERMUX_PKG_HOMEPAGE="https://jellyfin.org"
+TERMUX_PKG_DESCRIPTION="A free media system for organizing and streaming media (server)"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="10.10.7"
+_FFMPEG_VERSION="7.0.2-9"
+TERMUX_PKG_SRCURL=(
+	"https://github.com/jellyfin/jellyfin/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+	"https://github.com/jellyfin/jellyfin-ffmpeg/archive/refs/tags/v${_FFMPEG_VERSION}.tar.gz"
+)
+_GIT_URL=(
+	"git+https://github.com/ericsink/cb.git"
+	"git+https://github.com/ericsink/SQLitePCL.raw.git"
+	"git+https://github.com/mono/skia.git"
+	"git+https://github.com/jellyfin/jellyfin-web.git"
```

Comment:
- 1.) There shouldn't be `.git` suffixes on those URLs.
- 2.) Consider using associcative arrays (hashmaps) if you need a secondary index, e.g. the project name.
`kitty`'s build script does something similar to this.
https://github.com/termux/termux-packages/blob/c1dd8cccf917885f12ecb3b81e202787ee43c16a/x11-packages/kitty/build.sh#L34-L51